src/core/dune

```diff
@@ -16,6 +16,7 @@
    glob
    lib_parsing ; Parse_info.ml
    ast_generic
+   networking
```

Comment:
This is a bit unfortunate ... to depend on networking code in "the core".

Reply:
maybe add a comment next to the dependency saying it's just used for the Http_Request.t type in it which is used in Rule.ml for secrets.